lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -1543,29 +1536,30 @@ namespace Js
                 push eax; // push the return value into the stack
                 push layout;
                 call InterpreterStackFrame::GetDynamicRetType;
-                cmp eax, Void;
-                je end;
-                cmp eax, Signed;
-                je end;
-                cmp eax, Float;
-                jne skipFloat;
-                // float
+                mov ecx, eax;
```

Comment:
ah interesting, didn't realize that! had assumed it just needed to be a byteable register

---
In reply to: [75201861](https://github.com/Microsoft/ChakraCore/pull/1446#discussion_r75201861) [](ancestors = 75201861)